### PR TITLE
2021 fr - index translation

### PR DESCRIPTION
**Pull request by nhumblot**
Translation of the home page to french.

### CODE REVIEW

File: 2021/docs/index.fr.md [hunk 1]
```diff
@@ -1,83 +1,67 @@
...
+**A04:2021-Conception non sécurisée** est une nouvelle catégorie, avec un accent sur les défauts de conception. Si nous
+voulons ajouter des contrôles en amont, cela appel à avoir recours aux modèles de menaces, aux modèles et principes de
+conception sécurisés, et aux architectures de référence.
+
+**A05:2021-Mauvaise configuration de sécurité** gagne une place ; 90% des applications ont été testées sur des
+vulnérabilités de ce type. Avec des logiciels de plus en plus paramétrables, il n'est pas surprenant de voir cette
+catégorie prendre de l'ampleur. L'ancienne catégorie *XML Entités externes (XXE)* est incluse dans celle-ci.
+
+**A06:2021-Composants vulnérables et obsolètes** était précédemment nommée *Utilisation de Composants avec des
+Vulnérabilités Connues*. Elle se place deuxième de l'enquête auprès de l'industrie, mais pouvait également entrer dans
```
Comment (nhumblot) [inline 2021/docs/index.fr.md:39]: > enquête auprès de l'industrie

must change due to https://github.com/OWASP/Top10/pull/577

**Comment by sslHello** [comment]
@nhumblot I am sorry, there will be more changes until the version gets finalized...
Please let be know if you like to send an update to your branch or if I should commit this version.
Cheers/Mes meilleurs salutations
Torsten

**Comment by nhumblot** [comment]
@sslHello no worries, I will check the merged PRs and make a diff.

I thought you would me to translate a full page before merging. Good to know. :slightly_smiling_face:

If the PR is not in draft status (like it is right now), it means it is ok on my to merge it. After that I let you decide if you want to merge or not. :+1:

**Comment by sslHello** [comment]
@nhumblot OK, Merci/Thank you.

**Pull request merged**